tests/golden/check-copy-paste/expected.gold

```diff
@@ -0,0 +1,48 @@
+=== Possible copy/paste errors ===
```

Comment:
Maybe we can add to this test other verification errors, so the test includes how their outputs relate together.